### PR TITLE
core rc 1

### PR DESCRIPTION
**Pull request by aruniverse**

### CODE REVIEW

File: packages/apps/cdn-viewer/package.json [hunk 1]
```diff
@@ -5,26 +5,25 @@
...
-    "@itwin/core-i18n": "3.0.0-dev.116",
-    "@itwin/core-markup": "3.0.0-dev.116",
-    "@itwin/core-orbitgt": "3.0.0-dev.116",
-    "@itwin/core-quantity": "3.0.0-dev.116",
-    "@itwin/core-react": "3.0.0-dev.116",
-    "@itwin/presentation-common": "3.0.0-dev.116",
-    "@itwin/presentation-frontend": "3.0.0-dev.116",
-    "@itwin/webgl-compatibility": "3.0.0-dev.116",
+    "@bentley/imodelhub-client": "rc",
+    "@bentley/itwin-client": "rc",
```
Comment (calebmshafer) [inline packages/apps/cdn-viewer/package.json:9]: If you don't use this directly, can you try and drop it? I should have a PR later today to completely remove it.

**Comment by aruniverse** [comment]
Need to trim deps but everything should be working, lmk if you see any issues

**Review by kckst8 — APPROVED** [review]
LGTM

File: packages/apps/cdn-viewer/package.json (hunk 1 above)
Comment (aruniverse) [reply packages/apps/cdn-viewer/package.json:9]: going to continue trimming deps in another pr

**Pull request merged**